hazelcast/src/main/java/com/hazelcast/instance/impl/ClusterTopologyIntent.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.instance.impl;
+
+public enum ClusterTopologyIntent {
+    /**
+     * Hazelcast is not deployed in a managed context
+     */
+    NOT_IN_MANAGED_CONTEXT,
+    /**
+     * Unknown state, but in managed context (Kubernetes)
+     */
+    UNKNOWN,
```

Comment:
maybe rename it as IN_MANAGED_CONTEXT__UNKNOWN_STATE

Reply:
renamed to `IN_MANAGED_CONTEXT_UNKNOWN`